scripts/events_module/relationship/pregnancy_events.py

```diff
@@ -947,22 +946,18 @@ def get_kits(
                     change_relationship_values(
                         cats_from=[kit],
                         cats_to=[parent],
-                        platonic_like=kit_to_parent["platonic"],
-                        dislike=kit_to_parent["dislike"],
-                        admiration=kit_to_parent["admiration"],
-                        comfortable=kit_to_parent["comfortable"],
-                        jealousy=kit_to_parent["jealousy"],
-                        trust=kit_to_parent["trust"],
+                        like=kit_to_parent[RelValue.LIKE],
+                        respect=kit_to_parent[RelValue.RESPECT],
+                        comfort=kit_to_parent[RelValue.COMFORT],
+                        trust=kit_to_parent[RelValue.TRUST],
```

Comment:
```suggestion
                        **kit_to_parent,
```

should be on lines 949-952 in case gh is a dummy